src/components/recovery/RecoveryModal/index.tsx

```diff
@@ -46,7 +44,7 @@ export function _RecoveryModal({
 
   // Trigger modal
   useEffect(() => {
-    if (!isSidebarRoute) {
+    if (!isSidebarRoute && (!isOwner || !isGuardian)) {
```

Comment:
I think we need to split this condition or use an `||` because the InProgressCard is shown to non-owners and non-guardians. We have a test-case to cover that but it doesn't fail because we are not mocking `isSidebarRoute`.